### PR TITLE
Added sunniness icons on plantings/_thumbnail and show page

### PR DESCRIPTION
**Pull request by GabrielSandoval**
Sorry I messed up my last PR (#766). I had some issues when trying to sync my branch with the updated one.

### CODE REVIEW

**Comment by GabrielSandoval** [comment]
Hi @cesy and @maco

screen shot 2015-07-09 at 12 36 49 pm

I've added some tests, added the text  alongside the image, and added the alt text for each image.
Tell me if there's anything more needed to be done.

Thanks!

File: app/views/plantings/_thumbnail.html.haml [hunk 1]
```diff
@@ -16,7 +16,10 @@
       %dt Finished on:
       %dd= "#{display_finished(planting)}"
       %dt Sun/shade?:
-      %dd= "#{display_sunniness(planting)}"
+      %dd
+        - sunniness = planting.sunniness.blank? ? "not specified" : planting.sunniness
```
Comment (cesy) [inline app/views/plantings/_thumbnail.html.haml]: Does this need updating to use the new unspecified icon?

File: app/views/plantings/show.html.haml [hunk 2]
```diff
@@ -28,11 +28,13 @@
...
-      %p
-        %b 
-          = "Sun or shade?: "
-        = "#{display_sunniness(@planting)}"
+    %p
+      %b 
+        = "Sun or shade?: "
+      - sunniness = @planting.sunniness.blank? ? "not specified" : @planting.sunniness
+      = image_tag("sunniness_#{sunniness}.png", :size => "25x25", :alt => "#{sunniness}", :title => "#{sunniness}")
+      = " (#{sunniness})"
```
Comment (cesy) [inline app/views/plantings/show.html.haml]: Great, and once people are used to the changes, we can eventually decide if we want to keep the text or one day remove it.

**Comment by cesy** [comment]
Two of the images have transparent background and two are white background - can we make this consistent, please? I prefer the transparent background, personally.

File: app/views/plantings/_thumbnail.html.haml (hunk 1 above)
Comment (GabrielSandoval) [inline app/views/plantings/_thumbnail.html.haml]: Hi cesy. This is all good. The file name of the icon for unspecified sunniness is "sunniness_not specified.png". The 'sunniness' variable will decide which icon will be shown. In the case of a blanked sunniness, the 'sunniness' variable will contain 'not specified' and the image_tag will load the image 'sunniness_not specified.png'

Comment (cesy) [inline app/views/plantings/_thumbnail.html.haml]: Ah yes, got it, thanks.

**Comment by GabrielSandoval** [comment]
I made all the icons transparent. It looks better now. :+1:

**Comment by cesy** [comment]
Very nearly there, but it's come up with fresh merge conflicts - can you fix, please?

**Comment by pozorvlak** [comment]
The question mark isn't very visible on "not specified" - could you make it a bit larger/darker? Possibly by shortening the sunbeams to give more room inside the sun. And I think an even-weight font would match the style of the icon better.

**Comment by GabrielSandoval** [comment]
Will this do it?

screen shot 2015-07-10 at 12 27 56 pm

**Comment by GabrielSandoval** [comment]
Is this better?

screen shot 2015-07-10 at 1 08 51 pm

**Comment by pozorvlak** [comment]
Yes!

**Pull request merged**